modules/ROOT/pages/manage-versions-instances-concept.adoc

```diff
@@ -3,25 +3,24 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-After you create an API in Design Center for example, you set a version name and publish it to Exchange. In this release, the version resides in Exchange, not API Manager. You then set up API Manager to manage the version.
+After you create an API in Anypoint Paltform, you configure the version details for the API and then publish it to Exchange. The API version resides in Exchange, and not in API Manager. However, you then manage this API version from API Manager:
 
 image::environment2.png[]
 
-== API Instance Management in API Manager
+== Managing API Instances in API Manager
 
-In API Manager, you may also create, import, and deprecate API instances.
+You can create, import, and deprecate API instances using API Manager. You can use instances of an API in multiple environments. You can also have multiple instances of an API in the same environment:
 
-On the API administration dashboard, you expand the API version name to display the instances if there are any:
+//image::jsonplaceholder2.png[]
 
-image::jsonplaceholder2.png[]
+You can create one instance of an API that serves as a proxy and create another to directly manage as a basic endpoint. You can apply policies to the API based on how you have configured the endpint. For example, you can apply caching policies to the proxy, and throttling policies and security policies to the basic endpoint. 
```

Comment:
version?
```suggestion
You can create one instance of an API version that serves as a proxy and create another to manage directly as an endpoint. You can apply policies to the API based on how you configure that endpoint. For example, you can apply caching policies to the proxy, and throttling policies and security policies to the endpoint. 
```
The example is not an example of the sentence before it, which tells you that you can apply policies to the API based on how you configure the endpoint but does not mention applying policies to the API through the proxy.
It is unclear here which is the dominant/manager instance: the endpoint or the proxy.

Reply:
Fixed! Thanks!